(ForumsTeam) Update on Disk sizes and performance

vsubm opened the pull request:
Under the section: 
Default Azure disks
Operating system disk - Operating system disks can be sized up to 1 terabyte,and hosts the VMs operating system.

It has been updated to 4TB for Azure VMs
Blog link: https://azure.microsoft.com/en-in/blog/azure-introduces-new-disks-sizes-up-to-4tb/ 

Under the section: 
VMs disk types-> Premium disk performance

It's stated that Premium disks come in three types (P10, P20, P30), the size of the disk determines the disk type. When selecting, a disk size the value is rounded up to the next type. For example, if the size is below 128 GB the disk type will be P10, between 129 and 512 P20, and over 512 P30. 
Can we update that Large Disks are currently available in all Azure regions i.e P40(2TB) and P50(4TB) including IOPS Perdisk and Disk Bandwidth.
Blog Link: https://azure.microsoft.com/en-in/blog/azure-introduces-new-disks-sizes-up-to-4tb/

azurecla commented:
Hi __@abaranch__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

vsubm committed:
(ForumsTeam) Update on Disk sizes and performance

Under the section: 
Default Azure disks
Operating system disk - Operating system disks can be sized up to 1 terabyte,and hosts the VMs operating system.

It has been updateed to 4TB for Azure VMs
Blog link: https://azure.microsoft.com/en-in/blog/azure-introduces-new-disks-sizes-up-to-4tb/ 

Under the section: 
VM disk types-> Premium disk performance
It's stated that Premium disks come in three types (P10, P20, P30), the size of the disk determines the disk type. When selecting, a disk size the value is rounded up to the next type. For example, if the size is below 128 GB the disk type will be P10, between 129 and 512 P20, and over 512 P30. 
Can we update that Large Disks are currently available in all Azure regions i.e P40(2TB) and P50(4TB) including IOPS Perdisk and Disk Bandwidth.
Blog Link: https://azure.microsoft.com/en-in/blog/azure-introduces-new-disks-sizes-up-to-4tb/
